src/components/quickstarts/SupportSection.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css } from '@emotion/react';
+import { useTessen, Link } from '@newrelic/gatsby-theme-newrelic';
+import {
+  SUPPORT_LINK,
+  COMMUNITY_LINK,
+  QUICKSTART_SUPPORT_LEVELS,
+} from '../../data/constants';
+
+const SupportSection = ({ supportLevel }) => {
+  const tessen = useTessen();
+  const whatsIncluded = <strong>What's included</strong>;
+  const supportLink = (
+    <Link
+      to={SUPPORT_LINK}
+      key={QUICKSTART_SUPPORT_LEVELS.NEWRELIC}
+      onClick={() =>
+        tessen.track('instantObservability', 'QuickstartDetailsSupportClick')
+      }
+    >
+      Visit our Support Center
+    </Link>
+  );
+
+  const communityLink = (
+    <Link
+      to={COMMUNITY_LINK}
+      key={QUICKSTART_SUPPORT_LEVELS.NEWRELIC}
+      onClick={() =>
+        tessen.track('instantObservability', 'QuickstartDetailsCommunityClick')
+      }
+    >
+      the Explorers Hub
+    </Link>
+  );
+
+  const QUICKSTART_SUPPORT_CONTENT = {
+    [QUICKSTART_SUPPORT_LEVELS.NEWRELIC]: {
+      title: 'Built by New Relic',
+      content: [
+        'Need help? ',
+        supportLink,
+        ' or check out our community forum, ',
+        communityLink,
+        '.',
+      ],
```

Comment:
I think we could simplify this a bit by using string interpolation
```suggestion
      content: (<p>Need help? {supportLink} or check out our community forum, {communityLink}.</p>),
```